src/main/java/no/nav/pto/veilarbportefolje/arbeidsliste/v2/ArbeidsListeV2Controller.java

```diff
@@ -0,0 +1,152 @@
+package no.nav.pto.veilarbportefolje.arbeidsliste.v2;
+
+import io.vavr.control.Try;
+import io.vavr.control.Validation;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.common.types.identer.AktorId;
+import no.nav.common.types.identer.Fnr;
+import no.nav.pto.veilarbportefolje.arbeidsliste.Arbeidsliste;
+import no.nav.pto.veilarbportefolje.arbeidsliste.ArbeidslisteService;
+import no.nav.pto.veilarbportefolje.arbeidsliste.ArbeidslisteDTO;
+import no.nav.pto.veilarbportefolje.auth.AuthService;
+import no.nav.pto.veilarbportefolje.auth.AuthUtils;
+import no.nav.pto.veilarbportefolje.domene.AktorClient;
+import no.nav.pto.veilarbportefolje.domene.value.NavKontor;
+import no.nav.pto.veilarbportefolje.domene.value.VeilederId;
+import no.nav.pto.veilarbportefolje.service.BrukerServiceV2;
+import no.nav.pto.veilarbportefolje.util.ValideringsRegler;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.function.Function;
+
+import static no.nav.common.utils.StringUtils.nullOrEmpty;
+import static no.nav.pto.veilarbportefolje.util.SecureLog.secureLog;
+import static no.nav.pto.veilarbportefolje.util.ValideringsRegler.validerArbeidslisteV2;
+
+@Slf4j
+@RestController
+@RequestMapping("/api/v2")
+public class ArbeidsListeV2Controller {
```

Comment:
Jeg bare lurte på hvorfor også ArbeidslisteController skifter navn til ArbeidslisteV2Controller? Er det for å gjøre det enda mer "klart" eller kan det ikke være to klasser om heter det samme selv om de er på hhv v1 og v2?

Reply:
Siden de lever i to forskjellige pakker er det jo egentlig OK at de heter det samme, men ja er vel mest for lesbarhet :)